config.json

```diff
@@ -23,6 +23,17 @@
       "unlocked_by": null,
       "uuid": "74515d45-565b-4be2-96c4-77e58efa9257"
     },
+    {
+      "core": false,
+      "difficulty": 6,
+      "slug": "parallel-letter-frequency",
+      "topics": [
+        "concurrency",
+        "type_conversion"
```

Comment:
Maybe also `maps`? And maybe `strings`? And why `type_conversion`?

Reply:
@FridaTveit I'm happy to add those-- didn't know if I should just emphasize what made this different, or add more things and risk larding it up too much.

I put in `type_conversion` because of the way that `int`, `char`, and `Integer` are all use in the handling of the "letters".  But if you think it's not appropriate I'm happy to remove it, LMK.

Reply:
We've been adding topics that users might encounter when implementing the exercise. Topics that make this exercise different are possibly most important but I think more common topics like `strings` and `maps` are important as well. Adding them means that users who particularly want to practice using maps can look for all exercises with that topic and then do them. That's how I'm imagining the topics could be used anyway, I might be wrong :)

I would only add `type_conversion` if a typical solution required explicitly casting from one type to another. And this exercise seems to only require you to count the frequency of each letter which you could do without any type conversions? But if you think type conversion would be part of a good solution then I'm happy for it to stay in :)

Reply:
You're right-- my solution uses that technique, but it wouldn't be required.  It _is_ used in the tests, but still, I don't think the user would need to use it.

Reply:
I think these topics still need updating? :)

Reply:
Oops...I did that, honest!  I must have forgotten to commit the work.  Thanks.